server/server.js

```diff
@@ -15,4 +18,18 @@ server.on("listening", () => {
 
 process.on("SIGTERM", () => server.close(() => disconnectDb()));
 
+
+
+
+app.get("/hello", (req, res) => {
+	res.json({ message: "Hello World" });
+});
+
+// app.post("/", (req, res) => {
+// 	const insertQ = "INSERT INTO energizers(name,energizer) VALUES($1,$2)";
+// 	db.query(insertQ, [req.name, req.energizer]).then(() => {
+// 		res.status(201).json({ message: "table updated" });
+// 	});
+// });
```

Comment:
This commented out code should not be part of the pull request

Reply:
I have removed this part in my last commit before the pull request